Source/Train-Railway/Program.cs

```diff
@@ -85,92 +75,76 @@ static void InitializeTimeTable()
             FileManager.SplitFile(tmp, timeTables);
 
-            Console.WriteLine("TIMETABLE");
-            foreach (var t in timeTables)
-            {
-                Console.WriteLine($"{t.Arrival} {t.Departure} {t.StationID} {t.TrainID}");
-            }
+            //Console.WriteLine("TIMETABLE");
+            //foreach (var t in timeTables)
+            //{
+            //    Console.WriteLine($"{t.Arrival} {t.Departure} {t.StationID} {t.TrainID}");
+            //}
         }
 
         static void InitializePassengers()
         {
             string[] tmp = FileManager.ReadFile(FilesPath.PassengersPath);
             FileManager.SplitFile(tmp, passengers);
 
-            Console.WriteLine("\n\rPASSENGERS");
-            foreach (var t in passengers)
-            {
-                Console.WriteLine($"{t.ID} {t.Name}");
-            }
+            //Console.WriteLine("\n\rPASSENGERS");
+            //foreach (var t in passengers)
+            //{
+            //    Console.WriteLine($"{t.ID} {t.Name}");
+            //}
         }
 
         static void InitializeTrains()
         {
             string[] tmp = FileManager.ReadFile(FilesPath.TrainPath);
             FileManager.SplitFile(tmp, trains);
 
-            Console.WriteLine("\n\rTRAINS");
-            foreach (var t in trains)
-            {
-                Console.WriteLine($"{t.ID} {t.Name} {t.MaxSpeed} {t.Operated}");
-            }
+            //Console.WriteLine("\n\rTRAINS");
+            //foreach (var t in trains)
+            //{
+            //    Console.WriteLine($"{t.ID} {t.Name} {t.MaxSpeed} {t.Operated}");
+            //}
         }
 
         static void InitializeStations()
         {
             string[] tmp = FileManager.ReadFile(FilesPath.StationPath);
             FileManager.SplitFile(tmp, stations);
 
-            Console.WriteLine("\n\rSTATIONS");
-           /* foreach (var t in stations)
-            {
-                Console.WriteLine($"{t.ID} {t.StationName} {t.EndStation}");
-                t.IsAvaliable = true;
-            }*/
+            //Console.WriteLine("\n\rSTATIONS");
+            //foreach (var t in stations)
+            //{
+            //    Console.WriteLine($"{t.ID} {t.StationName} {t.EndStation}");
+            //    t.IsAvaliable = true;
+            //}
         }
 
         static void InitializeTracks()
         {
             string[] tmp = FileManager.ReadFile(FilesPath.TrainTrackPath);
             FileManager.SplitFile(tmp, tracks);
 
-            Console.WriteLine("\n\rSTATIONS");
-            foreach (var t in tracks)
-            {
-                Console.WriteLine($"{t.ID} {t.StartStation} {t.Distance} {t.EndStation}");
-            }
+            //Console.WriteLine("\n\rSTATIONS");
+            //foreach (var t in tracks)
+            //{
+            //    Console.WriteLine($"{t.ID} {t.StartStation} {t.Distance} {t.EndStation}");
+            //}
         }
 
         public static void StartTrain1()
```

Comment:
Think of Single responsibility principle (SRP). "A method holds logic for a single functionality or task. So, it reduces the code complexity. As each method has a single functionality so it is easy to read and maintain."